client/src/client.rs

```diff
@@ -1079,15 +1079,15 @@ impl Client {
         retry_in: Duration,
         pagination: Pagination,
     ) -> Result<Option<SignedTransaction>> {
-        let pagination: Vec<_> = pagination.into();
+        let pagination = pagination.into_query_parameters();
         for _ in 0..retry_count {
             let response = DefaultRequestBuilder::new(
                 HttpMethod::GET,
                 self.torii_url
                     .join(uri::PENDING_TRANSACTIONS)
                     .expect("Valid URI"),
             )
-            .params(pagination.clone())
```

Comment:
Afaik `.pagination` accepts anything which implements `IntoIter<Item = impl Borrow<(K, V)>>` do we even need to produce `Vec` for `Pagination` and `Sorting`? 

E.g `Sorting` can just return `Option` which is already implement `IntoIterator`.

Reply:
Changed to return impl IntoIterator for both Pagination and Sorting.